src/constants/ActionTypes.js

```diff
@@ -541,6 +543,7 @@ export default utils.deepKeyMirror({
 			REQUEST: null,
 		},
 		SET_ACTIVE_KEY: null,
+		SET_ACTIVE_KEYS: null,
```

Comment:
Does it make sense to set multiple scopes as active?

Reply:
Not much for me. Said. This is result of merging setActiveKey and setActiveKeys into one class SETTING_ACTIVE_ACTIONS. And yes, I think we should split it into two classes: SETTING_ACTIVE_KEYS_ACTIONS and SETTING_ACTIVE_KEY_ACTIONS where SETTING_ACTIVE_KEYS_ACTIONS will contain both actions setActiveKey and setActiveKeys.